src/app/shared/stache/modules/tutorial/tutorial-step.component.scss

```diff
@@ -0,0 +1,28 @@
+.stache-template-step {
+    border: 1px solid #dddddd;
+    margin: 20px 40px 70px 0px;
+    padding: 5px 50px 60px 50px;
+    text-align: center;
+    background-color: #ffffff;
+    position: relative;
```

Comment:
Hmm. So it looks like Lindsey doesn't do the center align on the items, just the title. So maybe we should just have the title center aligned rather than the entire step?